x-pack/metricbeat/module/aws/ec2/ec2.go

```diff
@@ -103,19 +107,35 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	}
 
 	awsConfig.Region = moduleConfig.DefaultRegion
-
 	svcEC2 := ec2.New(awsConfig)
 	regionsList, err := getRegions(svcEC2)
 	if err != nil {
 		err = errors.Wrap(err, "getRegions failed")
-		logp.Error(err)
+		ec2Logger.Error(err.Error())
+	}
+
+	//Calculate duration based on period
+	durationString, periodSec, err := convertPeriodToDuration(moduleConfig.Period)
+	if err != nil {
+		ec2Logger.Error(err.Error())
+	}
+
+	// Check if period is set to be multiple of 60s or 300s
+	remainder := periodSec % 300
```

Comment:
Comment says it checks for 60 or 300, but it only checks for 300. Also log message below is only 300.

Reply:
Yeah I'm also struggling here since 60s is for detailed monitoring and 300s is for basic monitoring. I changed some wording in the new commit. Please let me know if it looks better. Thanks!